<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>841</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MiniCAD</string>
  </property>
  <property name="windowIcon">
   <iconset resource="pic.qrc">
    <normaloff>:/resouce/AutoCAD.png</normaloff>:/resouce/AutoCAD.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Tab 1</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Tab 2</string>
       </attribute>
      </widget>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QWidget" name="widget" native="true">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>0</height>
       </size>
      </property>
      <widget class="QWidget" name="">
       <property name="geometry">
        <rect>
         <x>3</x>
         <y>13</y>
         <width>192</width>
         <height>461</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2" stretch="4,2,8">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,5">
            <item>
             <widget class="QLabel" name="label">
              <property name="minimumSize">
               <size>
                <width>50</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>50</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>画笔颜色:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="penColorBtn">
              <property name="minimumSize">
               <size>
                <width>120</width>
                <height>25</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>120</width>
                <height>25</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgb(255, 255, 255);
border: 1px solid black;
border-redius:2px;</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="4,7">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="minimumSize">
               <size>
                <width>50</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>50</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>画笔样式:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="penStyleCbox">
              <property name="minimumSize">
               <size>
                <width>120</width>
                <height>22</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>120</width>
                <height>22</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="2,7">
            <item>
             <widget class="QLabel" name="label_4">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>50</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>50</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="font">
               <font>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>X:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="xLineEdit">
              <property name="minimumSize">
               <size>
                <width>120</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>120</width>
                <height>16777215</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="2,0">
            <property name="rightMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_5">
              <property name="minimumSize">
               <size>
                <width>50</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>50</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="font">
               <font>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Y:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="yLineEdit">
              <property name="minimumSize">
               <size>
                <width>120</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>120</width>
                <height>16777215</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QTextEdit" name="NoteTextEdit">
          <property name="minimumSize">
           <size>
            <width>190</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>190</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>841</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionNew"/>
    <addaction name="actionSave"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuFigure">
    <property name="title">
     <string>Figure</string>
    </property>
    <addaction name="actionLine"/>
    <addaction name="actionRectangle"/>
    <addaction name="actionEllipse"/>
   </widget>
   <widget class="QMenu" name="menuSetting">
    <property name="title">
     <string>Setting</string>
    </property>
    <addaction name="actionMagnify"/>
    <addaction name="actionReduce"/>
    <addaction name="actionSelect"/>
    <addaction name="actionTrash"/>
   </widget>
   <widget class="QMenu" name="menuMeasure">
    <property name="title">
     <string>Measure</string>
    </property>
    <addaction name="actionVerDistance"/>
    <addaction name="actionHorDistance"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuFigure"/>
   <addaction name="menuSetting"/>
   <addaction name="menuMeasure"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>20</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="pic.qrc">
     <normaloff>:/resouce/OPEN.png</normaloff>:/resouce/OPEN.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="pic.qrc">
     <normaloff>:/resouce/save.png</normaloff>:/resouce/save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionLine">
   <property name="icon">
    <iconset resource="pic.qrc">
     <normaloff>:/resouce/Segment.png</normaloff>:/resouce/Segment.png</iconset>
   </property>
   <property name="text">
    <string>Line</string>
   </property>
  </action>
  <action name="actionRectangle">
   <property name="icon">
    <iconset resource="pic.qrc">
     <normaloff>:/resouce/rectangle.png</normaloff>:/resouce/rectangle.png</iconset>
   </property>
   <property name="text">
    <string>Rectangle</string>
   </property>
  </action>
  <action name="actionEllipse">
   <property name="icon">
    <iconset resource="pic.qrc">
     <normaloff>:/resouce/ellipse.png</normaloff>:/resouce/ellipse.png</iconset>
   </property>
   <property name="text">
    <string>Ellipse</string>
   </property>
  </action>
  <action name="actionPolygon">
   <property name="icon">
    <iconset resource="pic.qrc">
     <normaloff>:/resouce/polygon.png</normaloff>:/resouce/polygon.png</iconset>
   </property>
   <property name="text">
    <string>Polygon</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="pic.qrc">
     <normaloff>:/resouce/new-page.png</normaloff>:/resouce/new-page.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionMagnify">
   <property name="icon">
    <iconset resource="pic.qrc">
     <normaloff>:/resouce/magnify.png</normaloff>:/resouce/magnify.png</iconset>
   </property>
   <property name="text">
    <string>Magnify</string>
   </property>
  </action>
  <action name="actionReduce">
   <property name="icon">
    <iconset resource="pic.qrc">
     <normaloff>:/resouce/ic_search_reduce.png</normaloff>:/resouce/ic_search_reduce.png</iconset>
   </property>
   <property name="text">
    <string>Reduce</string>
   </property>
  </action>
  <action name="actionSelect">
   <property name="icon">
    <iconset resource="pic.qrc">
     <normaloff>:/resouce/Arrow.png</normaloff>:/resouce/Arrow.png</iconset>
   </property>
   <property name="text">
    <string>Select</string>
   </property>
  </action>
  <action name="actionTrash">
   <property name="icon">
    <iconset resource="pic.qrc">
     <normaloff>:/resouce/trash.png</normaloff>:/resouce/trash.png</iconset>
   </property>
   <property name="text">
    <string>Trash</string>
   </property>
  </action>
  <action name="actionVerDistance">
   <property name="icon">
    <iconset resource="pic.qrc">
     <normaloff>:/resouce/vertical-distance.png</normaloff>:/resouce/vertical-distance.png</iconset>
   </property>
   <property name="text">
    <string>VerDistance</string>
   </property>
  </action>
  <action name="actionHorDistance">
   <property name="icon">
    <iconset resource="pic.qrc">
     <normaloff>:/resouce/horizontal-distance.png</normaloff>:/resouce/horizontal-distance.png</iconset>
   </property>
   <property name="text">
    <string>HorDistance</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="pic.qrc"/>
 </resources>
 <connections/>
</ui>
